src/js/Content/Features/Store/Wishlist/FWishlistStats.js

```diff
@@ -139,7 +139,7 @@ export default class FWishlistStats extends Feature {
                     </div>`;
                 }).join("");
 
-                f.showDialog(wlStr.hidden, html);
+                f.showDialog(wlStr.hidden.toUpperCase(), html);
```

Comment:
I think it should be properly capitalized, but this is still better than before

Reply:
We should update the localization strings then, no idea why it was lower case to begin with 😄

Reply:
I guess that's better, yeah. I was just afraid of changing localization strings :D